src/main/java/org/prebid/server/events/EventRequest.java

```diff
@@ -30,9 +30,11 @@
 
     String lineItemId;
 
+    VastType vType;
+
     public enum Type {
 
-        win, imp
+        win, imp, VAST
```

Comment:
Rename all to UPPERCASE, or rename VAST to lowercase. Names of values in enum should have similar styling.